ci_scripts/make-test-tde.sh

```diff
@@ -3,6 +3,7 @@
 set -e
 
 export TDE_MODE=1
+export PERCONA_SERVER_VERSION=17.4.1
```

Comment:
I don't like this repeating.. With this, every time we update the version, we have to replace it at many places.

Can't we add it as a file called `PERCONA_SERVER_VERSION`, if we really want to do this?

Also, only the make scripts are updated, the meson scripts are left as-is.